couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -402,3 +408,66 @@ def extract_seconds_value(self, value):
             unit = 'us'
 
         return float(val) / TO_SECONDS[unit]
+
+    def _collect_index_stats_metrics(self):
+        url = urljoin(self._index_stats_url, INDEX_STATS_METRICS_PATH)
+        try:
+            data = self._get_stats(url)
+        except requests.exceptions.RequestException as e:
+            msg = "Error accessing the Index Statistics endpoint: %s: %s" % (url, str(e))
+            self.log.debug(str(e))
```

Comment:
```suggestion
            self.log.warning(str(e))
```
Can we log a warning instead?